poc/vidpf.py

```diff
@@ -6,7 +6,7 @@
 
 import hashlib
 
-from common import (format_dst, gen_rand, to_le_bytes, vec_add, vec_neg,
+from common import (Bytes, format_dst, gen_rand, to_le_bytes, vec_add, vec_neg,
```

Comment:
Prefer `bytes` instead of `Bytes`: https://github.com/cfrg/draft-irtf-cfrg-vdaf/issues/222

Reply:
Gotcha. This also needs to be changed in draft-irtf-cfrg-vdaf/poc/Field.py.

Reply:
If you have a sec, please send a PR :)